core/services/keeper/upkeep_executer_test.go

```diff
@@ -191,6 +191,58 @@ func Test_UpkeepExecuter_PerformsUpkeep_Happy(t *testing.T) {
 		txm.AssertExpectations(t)
 	})
 
+	// Copy of above test with GlobalEvmEIP1559DynamicFees set to true it will cause the upkeep executor to panic
+	t.Run("runs upkeep on triggering block number on EIP1559 chain GlobalEvmEIP1559DynamicFees true", func(t *testing.T) {
+		db, config, ethMock, executer, registry, upkeep, job, jpv2, txm, _, _, _ := setup(t)
+
+		config.Overrides.GlobalEvmEIP1559DynamicFees = null.BoolFrom(true)
```

Comment:
Do we have KeeperCheckUpkeepGasPriceFeatureEnabled = true by default?

Reply:
It is true by default. Just to be explicit.